systemtest/src/main/java/io/strimzi/systemtest/utils/StUtils.java

```diff
@@ -549,4 +549,19 @@ public static void waitUntilSupplierIsSatisfied(final BooleanSupplier sup) {
         TestUtils.waitFor(sup.getAsBoolean() + " is satisfied", TestConstants.GLOBAL_POLL_INTERVAL,
                 TestConstants.GLOBAL_STATUS_TIMEOUT, sup);
     }
+
+    /**
+     * Checks env variables of Topic operator container (inside EO Pod) and based on that determines, if BTO or UTO is used.
+     * Normally we can determine that based on {@link Environment#isUnidirectionalTopicOperatorEnabled()}, but in case that the
+     * FG is changed during test, we have no other way to check it.
+     *
+     * @param namespaceName name of the Namespace, where the EO Pod is running
+     * @param eoLabelSelector LabelSelector of EO
+     * @return boolean determining if UTO is used or not
+     */
+    public static boolean isUnidirectionalTopicOperatorUsed(String namespaceName, LabelSelector eoLabelSelector) {
+        return kubeClient().listPods(namespaceName, eoLabelSelector).get(0).getSpec().getContainers()
+            .stream().filter(container -> container.getName().equals("topic-operator")).findFirst().get().getEnv()
+            .stream().anyMatch(envVar -> envVar.getName().contains("ZOOKEEPER"));
```

Comment:
I guess this is due to fact that `ZOOKEEPER` is present only in bidirectional but could this be mentioned in comment ?

Reply:
good point, will do